tests/BenchmarkDotNet.Tests/Exporters/ApprovedFiles/ApprovalTest.Exporter.AsciiDocExporter_English (United States).approved.txt

```diff
@@ -0,0 +1,15 @@
+﻿....
```

Comment:
Could we rename the `*Invariant Language (Invariant Country)*` files to just `*Invariant*`?